src/rp2_common/pico_cyw43_driver/include/cyw43_configport.h

```diff
@@ -157,6 +157,10 @@ void cyw43_post_poll_hook(void);
 
 #define CYW43_POST_POLL_HOOK cyw43_post_poll_hook();
 
+// Allow malloc and free to be changed
+#define cyw43_malloc malloc
```

Comment:
maybe we should #ifndef these too, so users can change them - esp. in FreeRTOS malloc may not be the favorite thing